src/engraving/utests/beam_tests.cpp

```diff
@@ -225,3 +226,34 @@ TEST_F(BeamTests, flipBeamStemDir)
 
     delete score;
 }
+
+//---------------------------------------------------------
+//   flipTremoloStemDir
+//   This method tests if a tremolo's stem direction will be set to
+//   all its chords and will not affect other tremolos in score
+//   after using the flip command
+//---------------------------------------------------------
+
+TEST_F(BeamTests, flipTremoloStemDir)
+{
+    MasterScore* score = ScoreRW::readScore(BEAM_DATA_DIR + "flipTremoloStemDir.mscx");
+    EXPECT_TRUE(score);
+
+    Measure* m1 = score->firstMeasure();
+    ChordRest* cr = toChordRest(m1->findSegment(SegmentType::ChordRest, m1->tick())->element(0));
+    Tremolo* t = toChord(cr)->tremolo();
+    Chord* c1 = t->chord1();
+    Chord* c2 = t->chord2();
+    t->chord1()->up();
```

Comment:
Why is this here? This just return a bool whether its up or down. Seems pretty pointless. Did you forget to do something here? Maybe this could be the source of your problems?

Reply:
Good catch, but that isn't the source of the problem. The problem is that `t->up()` and `c1->up()` and `c2->up()` are all true, but after selecting the first chord and pressing x (executing `cmdFlip()` in other words) will flip `c1`, `c2`, and `t`. The line you identified can be safely removed, and I shall do that in the next push.